packages/server/providers/couchdb.js

```diff
@@ -0,0 +1,188 @@
+const listeners = {}
+let addListenersQueue = []
+
+const { localSettings, COUNTER_MESSAGE, dbSuffix } = require('../consts')
+
+const { COUCH_ACCOUNT, COUCH_URL, COUCH_PASSWORD } = process.env
+
+const couchDb = (app, io, pouchOptions) => {
+  const CouchDB = require('pouchdb-core')
+  const PouchDB = require('pouchdb-core')
+    .plugin(require('pouchdb-adapter-node-websql'))
+    .plugin(require('pouchdb-adapter-http'))
+    .plugin(require('pouchdb-mapreduce'))
+    .plugin(require('pouchdb-replication'))
+    .defaults(pouchOptions)
+  require('pouchdb-all-dbs')(PouchDB)
+
+  app.use('/db', require('express-pouchdb')(PouchDB))
+
+  const couchDbURL = (databaseName = '') => {
+    const dbWithoutSqlite = databaseName.replace(dbSuffix, '')
+    return `http://${COUCH_ACCOUNT}:${COUCH_PASSWORD}@${COUCH_URL}/${dbWithoutSqlite}`
+  }
+
+  // changesListener
+  const initChangesListener = (dbName) => {
+    const db = new CouchDB(couchDbURL(dbName))
+    // saving listener
+    listeners[dbName] = db.changes({
+      since: 'now',
+      live: true,
```

Comment:
I'm looking at the CouchDb docs: https://docs.couchdb.org/en/3.2.0/api/database/changes.html

I don't see the `live` parameter. Is it because we're using an older version?